readme.md

```diff
@@ -25,6 +25,27 @@ normalizeUrl('HTTP://xn--xample-hva.com:80/?b=bar&a=foo');
 ```
 
 
+## API
+
+### normalizeUrl(str, [options])
+
+#### str
+
+*Required*  
+Type: `string`
+
+#### options.keepProtocol
```

Comment:
From some more thinking this doesn't really reflect what it does. It prevents adding `http` protocol to protocol-relative URLs. Maybe `options.preserveProtocolRelative`. A mouthful, but much more descriptive. Thoughts?